### PR TITLE
[ROCKETMQ-218] README.md update

### PR DESCRIPTION
**Pull request by zhoudiqiu**
https://issues.apache.org/jira/browse/ROCKETMQ-218

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,12 +30,11 @@ It offers a variety of features as follows:
 
 ## Apache RocketMQ Community
 * [RocketMQ Community Incubator Projects](https://github.com/rocketmq)
-* [RocketMQ Community Projects](https://github.com/apache/incubator-rocketmq-externals)
```
Comment (zhoudiqiu) [inline README.md]: Requested by the team

File: README.md [hunk 2]
```diff
@@ -3,18 +3,18 @@
...
-* Long pull queue model,also support push consumption style
-* Million message accumulation ability in single queue
-* Over a variety of messaging protocols.such as JMS,MQTT etc.
-* Distributed high available deploy architecture, meets at least once message delivery semantics
-* Docker images for isolated testing and cloud Isolated clusters
-* Feature-rich administrative dashboard for configuration,metrics and monitoring
+* Long pull queue model and push consumption style
+* Million-level message accumulation capacity in single queue
+* Useful messaging protocols like [JMS](http://docs.oracle.com/javaee/6/tutorial/doc/bncdq.html)
+* Distributed highly available deployment architecture, meeting at least one message delivery semantic
```
Comment (zhouxinyu) [inline README.md]: `At-most-once, At-least-once, Exactly-once`, no need to replace `once` with `one`.

File: README.md [hunk 3]
```diff
@@ -3,18 +3,18 @@
...
+* Pub/Sub messaging model
 * Reliable FIFO and strict sequential messaging in the same queue
-* Long pull queue model,also support push consumption style
-* Million message accumulation ability in single queue
-* Over a variety of messaging protocols.such as JMS,MQTT etc.
-* Distributed high available deploy architecture, meets at least once message delivery semantics
-* Docker images for isolated testing and cloud Isolated clusters
-* Feature-rich administrative dashboard for configuration,metrics and monitoring
+* Long pull queue model and push consumption style
+* Million-level message accumulation capacity in single queue
```
Comment (lizhanhui) [inline README.md]: in single queue --> in a single queue

File: README.md (hunk 1 above)
Comment (lizhanhui) [inline README.md]: The external projects releases at their own paces. Links will be added to rocketmq web site instead.

**Comment by vongosling** [comment]
+1

**Pull request closed without merging**